client/src/components/util/graphql/Mutations.ts

```diff
@@ -47,6 +47,29 @@ export const UPDATE_ITEM = gql`
     }
 `;
 
+export const CREATE_REQUEST = gql`
+    mutation createRequest($newRequest: RequestInput!) {
+        createRequest(newRequest: $newRequest) {
+            request_id
+            user {
+                uuid
+                name
+                haveID
+            }
+            item {
```

Comment:
Make sure these item props match those in the hardware desk screen, they have to be the same for the caching to work (really we should be using GraphQL Fragments inside these queries, but I haven't done that)